site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-16-June-2022.adoc

```diff
@@ -0,0 +1,113 @@
+= Apache Cassandra Changelog #16
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: June 2, 2022
+:page-post-author: Chris Thornett
+:description: Apache Cassandra Changelog
+:keywords: Cassandra, changelog, updates, news, release notes, community,
+
+image::blog/changelog_header.jpg[Apache Cassandra Changelog]
+Our monthly roundup of key activities and knowledge to keep the community informed.
+
+== Release Notes
+
+=== Released
+
+In May, we hit our one-year branch and froze the code for the release of Apache Cassandra 4.1. If you would like to demo the new release for non-production use, a test build of https://dist.apache.org/repos/dist/release/cassandra/4.1-alpha1/[Cassandra 4.1-alpha1^] was approved on 27 May.
+
+Please read the https://gitbox.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=NEWS.txt;hb=refs/tags/4.1-alpha1-tentative[release notes^] and let us know if you encounter any problems.
```

Comment:
The link to the release notes is broken, should be: https://gitbox.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=NEWS.txt;hb=refs/tags/cassandra-4.1-alpha1